share/models/base.py

```diff
@@ -166,11 +144,11 @@ class Meta:
 class ShareObject(models.Model, metaclass=ShareObjectMeta):
     id = models.AutoField(primary_key=True)
     objects = ShareObjectManager()
-    versions = VersionManager()
     changes = fields.GenericRelationNoCascade('Change', related_query_name='share_objects', content_type_field='target_type', object_id_field='target_id', for_concrete_model=True)
 
     class Meta:
         abstract = True
+        base_manager_name = 'objects'
```

Comment:
Any reason to do this as rather than setting `_base_manager`?

Is this why all the other Metas inherit from this one?